frontend/src/app/store/app.store.ts

```diff
@@ -3,11 +3,12 @@ import { Task } from "../models/task";
 import { Injectable } from "@angular/core";
 import { BehaviorSubject, Observable } from "rxjs";
 import { TaskFilters } from "../models/task-filters";
+import { TaskStatus } from "../models/task-status";
 
 
 @Injectable({ providedIn: 'root' })
 export class AppStore {
-  private state: AppState = { filters: { status: null }, tasks: [] };
+  private state: AppState = { filters: { status: null,searchString: null }, tasks: [] };
```

Comment:
Don't forget to add space after `null,`.

Reply:
Done